src/pages/SyncProviderPage/SyncProviderPage.tsx

```diff
@@ -1,20 +1,17 @@
+import { useSuspenseQuery as useSuspenseQueryV5 } from '@tanstack/react-queryV5'
 import gt from 'lodash/gt'
 import isEmpty from 'lodash/isEmpty'
 import { Redirect } from 'react-router-dom'
 
-import { useSyncProviders } from 'services/config'
+import { SyncProvidersQueryOpts } from 'services/config/SyncProvidersQueryOpts'
 import { useInternalUser } from 'services/user/useInternalUser'
 import { loginProviderToShortName } from 'shared/utils/loginProviders'
 import A from 'ui/A'
 
 import SyncButton from './SyncButton'
 
-interface SyncButtonProps {
-  hasSynced: boolean
-}
-
-const SyncButtons: React.FC<SyncButtonProps> = ({ hasSynced }) => {
-  const { data: syncProviders } = useSyncProviders({ enabled: !hasSynced })
```

Comment:
do we lose any business logic by removing this `hasSynced` condition? Like the result of this operation is different now than before?

Reply:
No we we're not losing any business logic here, this was just being disabled, however the case where it would've been disabled the user is being redirected anyways so it won't be called.

Reply:
Got it, okay thank you for clarifying!